nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -82,6 +84,16 @@
             .identifiesControllerService(DBCPService.class)
             .build();
 
+    public static final PropertyDescriptor SQL_PRE_QUERY = new PropertyDescriptor.Builder()
+            .name("sql-pre-query")
+            .displayName("SQL Pre-Query")
+            .description("SQL pre-query to execute. Semicolon-delimited list of queries. "
+                    + "Note, the results/outputs of these queries will be suppressed if successfully executed.")
```

Comment:
You can probably omit "Note," here.

Reply:
Thanks for the suggestion, will omit it at next commit.